tools/osx/crosstool/BUILD.toolchains

```diff
@@ -10,76 +10,76 @@ OSX_TOOLS_CONSTRAINTS = {
     "armeabi-v7a": ["@platforms//cpu:arm"],
     "darwin_arm64": [
         "@platforms//os:osx",
-        "@platforms//cpu:aarch64",
+        "@platforms//cpu:arm64",
     ],
     "darwin_arm64e": [
         "@platforms//os:osx",
-        "@platforms//cpu:aarch64",
+        "@platforms//cpu:arm64",
     ],
     "darwin_x86_64": [
         "@platforms//os:osx",
         "@platforms//cpu:x86_64",
     ],
     "ios_arm64": [
         "@platforms//os:ios",
-        "@platforms//cpu:aarch64",
+        "@platforms//cpu:arm64",
     ],
     "ios_arm64e": [
         "@platforms//os:ios",
-        "@platforms//cpu:aarch64",
+        "@platforms//cpu:arm64",
     ],
     "ios_armv7": [
         "@platforms//os:ios",
-        "@platforms//cpu:arm",
+        "@platforms//cpu:armv7",
     ],
     "ios_i386": [
         "@platforms//os:ios",
-        "@platforms//cpu:x86_32",
+        "@platforms//cpu:i386",
     ],
     "ios_x86_64": [
         "@platforms//os:ios",
         "@platforms//cpu:x86_64",
     ],
     "ios_sim_arm64": [
         "@platforms//os:ios",
-        "@platforms//cpu:aarch64",
+        "@platforms//cpu:arm64",
     ],
     "tvos_arm64": [
-        "@platforms//os:ios",
-        "@platforms//cpu:aarch64",
+        "@platforms//os:tvos",
+        "@platforms//cpu:arm64",
     ],
     "tvos_x86_64": [
-        "@platforms//os:ios",
+        "@platforms//os:tvos",
         "@platforms//cpu:x86_64",
     ],
     "tvos_sim_arm64": [
-        "@platforms//os:ios",
-        "@platforms//cpu:aarch64",
+        "@platforms//os:tvos",
+        "@platforms//cpu:arm64",
     ],
```

Comment:
I wondered this in another context but is this one correct? since it's not actually different from `tvos_arm64`? maybe the name difference is enough?

Reply:
A while back we checked in environment constraints for simulator, device, and catalyst into https://github.com/bazelbuild/apple_support/blob/master/constraints/BUILD, but I don't know enough about Bazel's toolchains in a multi-repo situation to know if those would be accessible from here.

Reply:
ah yea, those wouldn't from here